frontend/sw360-portlet/src/main/webapp/html/components/includes/releases/editExternalIds.jsp

```diff
@@ -0,0 +1,59 @@
+<%@ page import="org.eclipse.sw360.portal.common.PortalConstants" %>
+<%@ page import="org.eclipse.sw360.datahandler.thrift.components.Release" %>
+<script src="<%=request.getContextPath()%>/webjars/jquery-validation/1.15.1/jquery.validate.min.js" type="text/javascript"></script>
+<script src="<%=request.getContextPath()%>/webjars/jquery-validation/1.15.1/additional-methods.min.js" type="text/javascript"></script>
+<table class="table info_table" id="externalIdsTable">
+    <thead>
+    <tr>
+        <th colspan="3" class="headlabel">External Ids</th>
+    </tr>
+    </thead>
+</table>
+
+<input type="button" class="addButton" onclick="addRowToExternalIdsTable();" value="Click to add row to External Ids "/>
+<br/>
+<br/>
+
+<script>
+
+    function deleteIDItem(rowIdOne) {
+        function deleteMapItemInternal() {
+            $('#' + rowIdOne).remove();
+        };
+
+        deleteConfirmed("Do you really want to remove this item?", deleteMapItemInternal);
+    }
+
+    function addRowToExternalIdsTable(key, value, rowId) {
+        if (!rowId) {
+            var rowId = "externalIdsTableRow" + Date.now();
+        }
+        if ((!key) || (!value)) {
+            var key = "", value = "";
```

Comment:
I don't know how bad that is, but if the key is null or undefined, this line will overwrite value with an empty string even if value was not empty, and vice versa.

Reply:
Yeah, you're right. I changed it to && and also added a small function to prevent people from creating external ids with one blank field. I think this fixes the problem?